client/src/components/pages/landing/info-block/info-block.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styles from './info-block.module.scss'
+import clsx from 'clsx'
+
+interface IInfoBlockProps {
+    color: avaliableColors;
+    header:string;
+    text:string;
+    picture: string;
+}
+type avaliableColors = 'cornflowerBlue' | 'whiteLilac'
+
+export const InfoBlock = ({color,header,text,picture}: IInfoBlockProps) => {
+	return (
+        <div className={clsx(styles.infoBlock, styles[color])}>
+            {color==='whiteLilac'?<img src={picture} alt="No connection"/>:null}
```

Comment:
this condition `color==='whiteLilac' `with passing `color` in props and using `styles[color]` sounds very complicated
what if tomorrow we will decide to change colors and names. Please, try to think of a more general solution.